	* setup.py: Fix handling of --without-threads.
	Already in Python 3.x

--- setup.py.old	2009-12-21 15:30:21.000000000 +0100
+++ setup.py	2009-12-21 15:31:39.000000000 +0100
@@ -1324,9 +1324,12 @@ class PyBuildExt(build_ext):
             if macros.get('HAVE_SEM_OPEN', False):
                 multiprocessing_srcs.append('_multiprocessing/semaphore.c')
 
-        exts.append ( Extension('_multiprocessing', multiprocessing_srcs,
-                                 define_macros=macros.items(),
-                                 include_dirs=["Modules/_multiprocessing"]))
+        if sysconfig.get_config_var('WITH_THREAD'):
+            exts.append ( Extension('_multiprocessing', multiprocessing_srcs,
+                                     define_macros=macros.items(),
+                                     include_dirs=["Modules/_multiprocessing"]))
+        else:
+            missing.append('_multiprocessing')
         # End multiprocessing
 
 
